src/devices/Bmxx80/Bme680.cs

```diff
@@ -307,70 +307,106 @@ public int GetMeasurementDuration(Bme680HeaterProfile profile)
             return (int)Math.Ceiling(measDuration);
         }
 
+        // TODO: what happens if temp measurement is skipped and humidity is read
         /// <summary>
-        /// Read the humidity.
+        /// Reads the humidity. A return value indicates whether the reading succeeded.
         /// </summary>
-        /// <returns>Calculated humidity.</returns>
-        public double ReadHumidity()
+        /// <param name="humidity">
+        /// Contains the measured humidity as %rH if the <see cref="HumiditySampling"/> was not set to <see cref="Sampling.Skipped"/>.
+        /// Contains <see cref="double.NaN"/> otherwise.
```

Comment:
ditto

Reply:
Other than the forgotten Todo am I missing something? The out parameter will still contain NaN if the measurement was skipped.

Reply:
`Contains <see cref="double.NaN"/> otherwise.`

Reply:
I see, I'd expect the call to return false when you're skipping but I'm fine either way

Reply:
Yeah the call will return false and the out parameter contains NaN if the measurement was skipped